circle.yml

```diff
@@ -1200,7 +1201,7 @@ jobs:
           command: yarn workspace @cypress/vue build
       - run:
           name: Run tests
-          command: yarn test --reporter cypress-circleci-reporter --reporter-options resultsDir=./test_results
```

Comment:
Can we keep the reporter options? I am not sure but I think it has something to do with formatting the output on CI.

Reply:
They are kept, they simply are specified through execa